example/src/App.js

```diff
@@ -1,19 +1,70 @@
 import React, { Component } from 'react';
 
-import { Button } from 'test-ui-lib';
+import { DropDown } from 'test-ui-lib';
```

Comment:
Around this whole file, are you deliberately changing the example app? I know you were playing about but don't we do all this stuff in stories?

Reply:
Yeah, I was just playing with it and didn't want to revert it, so that the next time it will be in the state I left it at. Not really important though (it is only us who will ever see it) so happy to revert it you'd like?

Reply:
I'm happy for it to stay, like you said has no impact